api/controllers/error_handling.controller.js

```diff
@@ -13,6 +13,8 @@ exports.handleCustomErrors = (err, req, res, next) => {
 exports.handlePSQLErrors = (err, req, res, next) => {
    if (err.code === "22P02") {
     res.status(400).send({ msg: "Invalid ID must be a number" });
+  } else if (err.code === "23503") {
+      res.status(404).send({ msg: "User does not exist" });
```

Comment:
this is fine for now but this `23503` won't be specifically for Users so this might either need to be made more generic ("resource doesn't exist"?) or you'll need to include more logic to specify what couldn't be found